docs/roman/references_general/distortion_reffile.rst

```diff
@@ -0,0 +1,45 @@
+:orphan:
+
+.. _distortion_reffile:
+
+DISTORTION Reference File
+-------------------------
+
+:REFTYPE: DISTORTION
+:Data model: `~roman_datamodels.datamodels.DistortionRefModel`
+
+The distortion reference file contains a combination of astropy models,
+representing the transform from detector to the telescope V2, V3 system.
+The following convention was adopted:
+
+- The output in the V2, V3 system is in units of arcsec.
+- The input x and y are 0-based coordinates in the DMS system.
+- The center of the first pixel is (0, 0), so the first pixel goes from -0.5 to 0.5.
+- The origin of the transform is taken to be (0, 0).
+  Note, that while a different origin can be used for some transforms the relevant
+  offset should first be prepended to the distortion transform to account for the change
+  in origin of the coordinate frame.  
+
+Internally the WCS pipeline works with 0-based coordinates.
+
+The model is a combination of polynomials.
```

Comment:
Remove this one too - unknown yet.

Reply:
maybe alter this to say,
so the coordinate range for the first pixel goes from -0.5 to 0.5.

Reply:
Removed.